hibernate-reactive-core/src/main/java/org/hibernate/reactive/adaptor/impl/ResultSetAdaptor.java

```diff
@@ -186,6 +187,16 @@ public String getString(String columnLabel) {
 
 	@Override
 	public boolean getBoolean(String columnLabel) {
+		final Object value = row.getValue( columnLabel );
+		wasNull = value == null;
+		if ( wasNull ) {
+			return false;
+		}
+
+		if( value instanceof BigInteger ) {
+			return ((BigInteger)value).intValue() > 0;
+		}
+
```

Comment:
uuhhhhh what's going on here?

Reply:
'twas an attempt to convert to boolean assuming the value is 1 or zero (true or false).  In the test value = "1"

Reply:
OK, but ordinarily it's the job of the Hibernate `Type`s to handle this sort of translation. Where a `boolean` is mapped to a numeric column, `NumericBooleanType` deals with this.